specs/qm/quality-management-shapes.html

```diff
@@ -303,53 +303,5 @@ <h3 id="resource_testexecutionrecord">Resource: TestExecutionRecordShape</h3>
         ></div>
       </section>
     </section>
-
-    <section class="appendix informative" id="history">
-      <h2>Change History</h2>
-      <table border="1" cellspacing="0" cellpadding="0">
-        <tr>
-          <td
-            width="103"
-            valign="top"
-            style="width: 77.4pt; border: solid windowtext 1pt; padding: 0in 5.4pt 0in 5.4pt;"
-          >
-            <b>Revision</b>
-          </td>
-          <td
-            width="96"
-            valign="top"
-            style="width: 1in; border: solid windowtext 1pt; border-left: none; padding: 0in 5.4pt 0in 5.4pt;"
-          >
-            <b>Date</b>
-          </td>
-          <td
-            width="144"
-            valign="top"
-            style="width: 1.5in; border: solid windowtext 1pt; border-left: none; padding: 0in 5.4pt 0in 5.4pt;"
-          >
-            <b>Editor</b>
-          </td>
-          <td
-            width="295"
-            valign="top"
-            style="width: 221.4pt; border: solid windowtext 1pt; border-left: none; padding: 0in 5.4pt 0in 5.4pt;"
-          >
-            <b>Changes Made</b>
-          </td>
-        </tr>
-        <tr>
-          <td>01</td>
-          <td>07/06/2016</td>
-          <td>Jim Amsden</td>
-          <td>Initial CSPRD01</td>
-        </tr>
-        <tr>
-          <td>02</td>
-          <td>08/08/2018</td>
-          <td>Gray Bachelor</td>
-          <td>Minor edits and checks for V2.1</td>
```

Comment:
@jamsden I have combined both entries in the new table because CSPRD01 was never published.